src/components/NewLocationPage/ChartFooter/MetricChartFooter.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import {
+  Footer,
+  ButtonContainer,
+  FooterText,
+  AboutText,
+  ModalButton,
+} from './ChartFooter.style';
+import { useBreakpoint } from 'common/hooks';
+import NewDialog from 'components/NewDialog/NewDialog';
+import { Link } from 'cms-content/modals';
+
+export interface AboutModalProps {
+  header: string;
+  body: string;
+  links: Link[];
+}
+
+const MetricChartFooter: React.FC<{
+  footerText: string | null;
+  shareButton: any;
+  aboutModal?: AboutModalProps;
+}> = ({ footerText, shareButton, aboutModal }) => {
+  const isMobile = useBreakpoint(600);
+  const [openModal, setOpenModal] = useState(false);
+  return (
+    <>
+      <Footer>
+        <FooterText>
+          {footerText}&nbsp;
+          {!isMobile && aboutModal && (
```

Comment:
instead of rendering things with isMobile and the useBreakpoint hook, see the uses of [MobileOnly](https://github.com/covid-projections/covid-projections/blob/develop/src/components/NewLocationPage/Shared/Shared.style.tsx#L53) and [DesktopOnly](https://github.com/covid-projections/covid-projections/blob/develop/src/components/NewLocationPage/Shared/Shared.style.tsx#L57) components